Framework/PythonInterface/test/python/mantid/api/AlgorithmTest.py

```diff
@@ -5,6 +5,8 @@
 from mantid.api import AlgorithmID, AlgorithmManager
 from testhelpers import run_algorithm
 
+if six.PY3:
+    unicode = str
```

Comment:
Why do you define `unicode` for `PY3` when the only use of `unicode` is hidden behind an `if six.PY2`?